.github/workflows/blank.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Run the build process with Docker
         uses: addnab/docker-run-action@v3
         with:
-            image: ghcr.io/janisz/indexer:goland-2022.2.4
+            image: ghcr.io/janisz/indexer:goland-2023.1.2
```

Comment:
Any plans to move this to the GHCR of this organization / repo (tbh, whatever this docker image does 🥴 )

Reply:
Indeed we could move it to stackrox https://github.com/janisz/jetbrains-indexer